### PR TITLE
[Config Linux] Use MUST and MAY for weight and leafWeight

### PR DESCRIPTION
**Pull request by RobDolinMS**
Signed-off-by: Rob Dolin

Signed-off-by: Rob Dolin <robdolin@microsoft.com>

### CODE REVIEW

File: config-linux.md [hunk 1]
```diff
@@ -361,7 +361,7 @@ The following parameters can be specified to setup the controller:
     * **`weight`** *(uint16, OPTIONAL)* - bandwidth rate for the device, range is from 10 to 1000
     * **`leafWeight`** *(uint16, OPTIONAL)* - bandwidth rate for the device while competing with the cgroup's child cgroups, range is from 10 to 1000, CFQ scheduler only
 
-    You must specify at least one of `weight` or `leafWeight` in a given entry, and can specify both.
+    You MUST specify at least one of `weight` or `leafWeight` in a given entry, and can specify both.
```
Comment (wking) [inline config-linux.md]: While we're cleaning up this line, maybe “can” → “MAY”?

**Comment by wking** [comment]
The initial 9388832 is missing an email from the Signed-off-by [1].
But it's in a branch in this repo.  Maybe someone with push access can
squash the two commits into one and keep the working Signed-off-by?

[1]: https://pullapprove.com/opencontainers/runtime-spec/pull-request/728/requirements/

**Comment by mrunalp** [comment]
@RobDolinMS you shouldn't push branches to origin directly. You can push branches to your fork and then create PRs from there. Thanks!

**Comment by wking** [comment]
On Wed, Mar 22, 2017 at 10:48:27AM -0700, Mrunal Patel wrote:
> @RobDolinMS you shouldn't push branches to origin directly. You can
> push branches to your fork and then create PRs from there. Thanks!

One way to accomplish this would be to remove @RobDolinMS' write
access for this repository.  If we still want maintainers to help
polish the PR (since @RobDolinMS has had trouble rebasing in the
past), we can still do that if he allows maintainers to edit his PR
branches [1].

[1]: https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

**Comment by hqhq** [comment]
I've carried this in #751 .

**Pull request closed without merging**